src/client/opamCommands.ml

```diff
@@ -2565,7 +2565,7 @@ let with_repos_rt gt repos f =
           | (n, Some url) ->
             let repo =
               OpamStd.Option.Op.(
-                OpamUrl.parse_opt ~handle_suffix:false ~from_file:false url
+                OpamUrl.parse_opt ~from_file:false url
```

Comment:
cf. https://github.com/ocaml/opam/issues/4673#issuecomment-872888360 - this ought to be CLI-version guarded, but that shouldn’t be too bad, as the version check _is_ then the value passed as `handle_suffix`